pdf/invoice.total-tab.tpl

```diff
@@ -28,7 +28,7 @@
     <td class="grey" width="50%">
       {l s='Total Products' d='Shop.Pdf' pdf='true'}
     </td>
-    <td class="white" width="50%">
+    <td class="white" style="text-align: right !important;" width="50%">
```

Comment:
It is suspicious that you added (then changed behavior ?) here a hard coded `text-align: right` instead of `class= ... text-end".

If there is a reason, maybe a comment would help some ignorant contributor finding this :thinking: